src/plugin/minimap.js

```diff
@@ -124,7 +124,7 @@ export default class MinimapPlugin {
             if (!document.body.contains(this.params.container)) {
                 ws.container.insertBefore(this.params.container, null);
             }
-
+            // how solve this dependency??
```

Comment:
Here, checking if there is `regions` plugin with `this.wavesurfer.regions`, forces to call the plugin "regions". In this way, extending regions plugin, it cannot be called with a custom name. 
Any idea of how to solve this "name" dependency ?